src/rules/prefer_primordials.rs

```diff
@@ -637,6 +670,17 @@ const noop = Function.prototype;
           hint: PreferPrimordialsHint::GlobalIntrinsic,
         },
       ],
+      r#"
+const { ArrayBuffer } = primordials;
+new ArrayBuffer(10).byteLength;
+      "#: [
+        {
+          line: 3,
+          col: 0,
+          message: PreferPrimordialsMessage::GlobalIntrinsic,
+          hint: PreferPrimordialsHint::GlobalIntrinsic,
+        },
+      ],
```

Comment:
Could you also add other test cases to check all the getters included in `GETTER_TARGETS` other than `byteLength`? I think it should be helpful to make sure everything is working

Reply:
Done!